gitops-tools/kpt-argocd-cmp/kpt-render/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG BUILDER_IMAGE=golang:1.23.5
+ARG BASE_IMAGE=quay.io/argoproj/argocd:v3.0.6
+
+FROM --platform=$BUILDPLATFORM ${BUILDER_IMAGE} AS build
+
+ENV CGO_ENABLED=0
+ENV GO111MODULE=on
+
+WORKDIR /go/src/kpt-render
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
```

Comment:
Do we need to copy everything? On the top we are doing selective copy so here we can do that as well